python/lsst/ts/Dome/dome_csc.py

```diff
@@ -362,29 +371,37 @@ async def config_llcs(self, data):
 
         Parameters
         ----------
-        data : `TBD`
-            The contents of this parameter will be defined soon.
+        data : `dict`
+            A dictinary with arguments to the function call. It should contain keys for all lower level
+            components to be configured with values that are dicts with keys for all the parameters that
+            need to be configured. The structure is
+            "AMCS":
+                "jmax"
+                "amax"
+                "vmax"
+            "LWSCS":
+                "jmax"
+                "amax"
+                "vmax"
+
+        It is assumed that the configuration configuration_parameters is presented as a dictionary of
```

Comment:
Can you reword to eliminate the stutter in "the configuration configuration_parameters"?

A simple example would help clarify. I *think* you are saying that the user can provide a subset of the above values, e.g.:
```
{"AMCS": {"amax": 5, "jmax": 4}}
```

Reply:
Good suggestion, thanks.